libeos-updater-util/avahi-service-file.h

```diff
@@ -44,4 +44,241 @@ gboolean eos_avahi_service_file_delete (const gchar   *avahi_service_directory,
                                         GCancellable  *cancellable,
                                         GError       **error);
 
+/* new DNS-SD records format for ostree */
+/* TODO: gtkdocize it? */
```

Comment:
Drop this `TODO`. As long as the content’s there, there’s no need to gtkdocize it yet.